content/en/docs/components/serving/kfserving.md

```diff
@@ -18,14 +18,14 @@ You can use KFServing to do the following:
 
 * Enable a simple, pluggable, and complete story for your production ML inference server by providing prediction, pre-processing, post-processing and explainability out of the box.
 
-Our strong community contributions help KFServing to grow. We have a Technical Steering Committee driven by Google, IBM, Microsoft, Seldon, and Bloomberg. [Browse the KFServing GitHub repo](https://github.com/kubeflow/kfserving) to give us feedback!
+Our strong community contributions help KFServing to grow. We have a Technical Steering Committee driven by AWS, IBM, Nvidia, Seldon, and Bloomberg. [Browse the KFServing GitHub repo](https://github.com/kubeflow/kfserving) to give us feedback!
```

Comment:
Just a suggestion - official unofficial names maybe:

```suggestion
Our strong community contributions help KFServing to grow. We have a Technical Steering Committee driven by Amazon Web Services (AWS), IBM Cloud, NVIDIA, Seldon, and Bloomberg. [Browse the KFServing GitHub repo](https://github.com/kubeflow/kfserving) to give us feedback!
```

(Had to go [here](https://s22.q4cdn.com/364334381/files/doc_financials/2020/ar/2020-nvidia-annualreport-content-r25-web-144dpi-combined.pdf) to check "NVIDIA" 🤣 Their site uses all caps everywhere, I think)